packages/esm-patient-tests-app/src/test-results/panel-view/panel-view.component.test.tsx

```diff
@@ -0,0 +1,216 @@
+import React from 'react';
+import { render, screen, fireEvent, within } from '@testing-library/react';
+import { useTranslation } from 'react-i18next';
+import PanelView from './panel-view.component';
+import usePanelData from './usePanelData';
+import classNames from 'classnames';
+
+jest.mock('@openmrs/esm-framework', () => ({
+  useLayoutType: jest.fn(() => 'desktop'),
+  navigate: jest.fn(),
+  isDesktop: jest.fn(() => true),
+  usePatient: jest.fn(() => ({ patientUuid: 'test-patient' })),
+  formatDate: jest.fn((date) => date.toISOString()),
+  restBaseUrl: '/ws/rest/v1',
+  createGlobalStore: jest.fn(() => ({
+    getState: jest.fn(() => ({ navGroups: [] })),
+    subscribe: jest.fn(),
+    unsubscribe: jest.fn(),
+    setState: jest.fn(),
+  })),
+  createUseStore: jest.fn(() => () => ({ navGroups: [] })),
+}));
+
+jest.mock('classnames', () => ({
+  __esModule: true,
+  default: jest.fn((...args) => args.join(' ')),
+}));
+
+jest.mock('@openmrs/esm-patient-common-lib', () => ({
+  EmptyState: jest.fn(({ headerTitle }) => <div data-testid="empty-state">{headerTitle}</div>),
```

Comment:
dtto, you can mock an empty dataset and return the `<EmptyState>` container just like we do it in other tests